components/BlocksTable/BlocksTable.tsx

```diff
@@ -57,16 +57,26 @@ const BlocksTable: FC<BlocksTableProps> = props => {
   const router = useRouter()
 
   return (
-    <CommonTable
-      {...props}
-      columns={COLUMNS}
-      onRowClick={(block: BlockType) =>
-        router.push({
-          pathname: RoutePaths.BlockInfo,
-          query: { id: block?.sequence.toString() },
-        })
-      }
-    />
+    // TODO: can remove container div once the Phase 3 badge is gone
+    // (https://airtable.com/appIXmGgVqP9QdbCf/tblAWSTOQbTq6vh6X/viwVMBWs7bGdpLiOd/reczg8YuO3jIBQ3pj)
+    <div>
```

Comment:
in react this should be a fragment, which is actually compiled out in most cases, and change the end div to </> too.

```suggestion
    <>
```